src/libraries/Microsoft.PowerFx.Interpreter/ReflectionFunction.cs

```diff
@@ -184,8 +184,9 @@ protected ReflectionFunction(string name, FormulaType returnType, params Formula
                 Name = name,
                 RetType = returnType,
                 ParamTypes = paramTypes,
-                _method = m
-            };
+                _method = m,
+                _isAsync = m.ReturnType.BaseType == typeof(Task)
```

Comment:
>_isAsync [](http://example.com/codeflow?start=16&length=8)

Just noticed we're also missing _configType.

Maybe we should make FunctionDescr have a ctor 
and make these properties private  - to enforce they're getting set in both places.

Reply:
updated to use Ctor